Scouting/src/main/java/ca/team2706/scouting/mcmergemanager/backend/FileUtils.java

```diff
@@ -834,4 +828,108 @@ public static void syncFiles(Context context) {
         getMatchesFromServer(context);
     }
 
+    // Checks to see if a file with a certain filename exists
+    public static boolean fileExists(Context context, String filename) {
+        File file = new File(context.getFilesDir(), filename);
+
+        if (file == null || !file.exists())
+            return false;
+        return true;
+    }
+
+    public static final String EVENT_KEYS_FILENAME = "EventKeys2017.json";
+
+    // Saves a list of all the events for a certain year,
+    // User then chooses what event that they are at
+    public static void getEventListAndSave(final int year, final Context context) {
```

Comment:
Maybe I'm missing it, but is there an equivalent method somewhere to read the event list from file (like, if the phone doesn't have internet)?

Reply:
Oh I see, you're doing it in the SettingsActivity. Clever :+1:

Reply:
Moved it.